### DISCUSSION TITLE
Manually trigger or simulate pointer events

### DISCUSSION THREAD

**Discussion by mackski** [nolimits4web/atropos#24] [General]
Is it possible to trigger animations or simulate pointer events for use with waypoints?

**Comment by Stephcraft** [comment]
You can try something like:
```js
atroposElement.dispatchEvent(new PointerEvent('pointermove', { pointerType: 'mouse', clientX: x, clientY: y }))
```
Or copy some of the logic from the source code and modifying `style` in a `setInterval`
